Runigram.java

```diff
@@ -60,48 +89,86 @@ private static void print(Color c) {
 	// For example, to check that some image processing function works correctly,
 	// we can apply the function and then use this function to print the resulting image.
 	private static void print(Color[][] image) {
-		//// Replace this comment with your code
+		for(int i = 0; i < image.length; i++){
+			for(int j = 0; j < image[0].length; j++){
+				System.out.print(image[i][j]);

```

Comment:
You can not print Object Color with `System.out.println`, use it own print method the exposed at this code.
```suggestion
				print(image[i][j]);

```